### PR TITLE
Include a new asciidoctor feature

### PR DESCRIPTION
**Pull request by jagedn**
As discussed in #164 this is a new feature to generate asciidoctor infraestructure when start a new project

### CODE REVIEW

File: base/features/asciidoctor/skeleton/gradle-build/src/docs/asciidoc/index.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+= The Title
+author@email.email
+
+Write a short introduction of your project
+
+== Subtitle
+
+You can orgnaize the documentation into section
```
Comment (sdelamo) [inline base/features/asciidoctor/skeleton/gradle-build/src/docs/asciidoc/index.adoc]: There is a typo in `orgnaize`

File: base/features/asciidoctor/skeleton/maven-build/src/docs/asciidoc/index.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+= The Title
+author@email.email
+
+Write a short introduction of your project
+
+== Subtitle
+
+You can orgnaize the documentation into section
```
Comment (sdelamo) [inline base/features/asciidoctor/skeleton/maven-build/src/docs/asciidoc/index.adoc]: There is a typo in `orgnaize`

**Comment by graemerocher** [comment]
The `index.adoc` doesn't need to be duplicated for both Maven and Gradle. It can go in a shared `skeleton` directory. Such as https://github.com/micronaut-projects/micronaut-profiles/tree/master/base/features/logback/skeleton/src/main/resources

File: base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle [hunk 3]
```diff
@@ -0,0 +1,24 @@
+asciidoctorj {
+    version '2.0.0'
+    modules{
+        diagram{
+            version '1.5.16'
```
Comment (abelsromero) [inline base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle]: Last available version is 1.5.18

File: base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    attributes 'sourcedir': 'src/docs/asciidoc',
+            'source-highlighter': 'coderay',
+            toc: 'left',
+            idprefix: '',
+            idseparator: '-',
+            'icons': 'font',
+            'setanchors': '',
+            'listing-caption':'',
+            'imagesdir': 'images',
```
Comment (abelsromero) [inline base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle]: I think only attributes with '-' need to be surrounded by quotes for Groovy syntax.
Also some attributes below use double quotes. Imho :% for consistency, all should use the same syntax.

File: base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle (hunk 3 above)
Comment (jagedn) [reply base/features/asciidoctor/skeleton/gradle-build/gradle/asciidoc.gradle]: Thanks @abelsromero. By other hand in gradle I'm using the plugin 3.0.0-alpha.2 althought alpha.3 is available. I tested both of them and for basic docs they work fine but my intent is to upgrade the profile when we'll have a release of the plugin

**Comment by graemerocher** [comment]
Thanks for the contribution!

**Pull request merged**